pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <hazelcast.version>4.2.1</hazelcast.version>
+        <hazelcast.version>5.2021.8</hazelcast.version>
```

Comment:
Having a non-released version in the master branch is not good. Maybe we can merge this into a maintenance branch (such as `5.0-snapshot`) and merge it with `master` when Hazelcast 5.0 is released.

Reply:
thanks :) created `5.0-snapshot` branch and changed base branch of this PR